src/vdaf/poplar1.rs

```diff
@@ -76,12 +83,14 @@ impl IdpfInput {
 
     /// Construct a new input that is a prefix of `self`. Bounds checking is performed by the
     /// caller.
+    #[cfg(feature = "internal_prg")]
```

Comment:
Poplar1 needs a lot of work, so minimally invasive change here is probably best. Please move these fences up to the module.

Reply:
This sounds like a good idea to me, provided that `poplar1` remains available with the crate's default features.